tools/docker/dev_container.Dockerfile

```diff
@@ -1,5 +1,12 @@
 #syntax=docker/dockerfile:1.1.5-experimental
-FROM addons_base:latest as build_dev_container
+FROM tensorflow/tensorflow:2.1.0-custom-op-ubuntu16 as dev_container_cpu
+ARG TF_PACKAGE
+ARG TF_VERSION
+
+# Temporary until custom-op container is updated
+RUN ln -sf /usr/bin/python3 /usr/bin/python
+RUN ln -sf /usr/local/bin/pip3 /usr/local/bin/pip
+RUN pip install --default-timeout=1000 $TF_PACKAGE==$TF_VERSION
```

Comment:
With this we are duplicating the tensorflow install layer.  Do you think to remove it after https://github.com/tensorflow/tensorflow/pull/40189 or need we to duplicate it for other goals?

Reply:
Yeah remove it after that is merged and a new container is published

Reply:
Ok I think that this for cpu could be removed now